autoload/neomake/makers/ft/elixir.vim

```diff
@@ -1,7 +1,11 @@
 " vim: ts=4 sw=4 et
 
+function s:find_mix_file(path)
```

Comment:
To make the linter happy, please add `abort` to the end of each `:function` line.